components/automate-ui/src/app/entities/users/user.reducer.ts

```diff
@@ -56,22 +56,40 @@ export function userEntityReducer(state: UserEntityState = UserEntityInitialStat
     case UserActionTypes.GET_FAILURE:
       return set('getStatus', EntityStatus.loadingFailure, state);
 
-    case UserActionTypes.UPDATE:
+    case UserActionTypes.UPDATE_PASSWORD_USER:
       return set('updateStatus', EntityStatus.loading, state);
```

Comment:
I did not have a reason to break apart the status into name and password. The UI currently response the same to both.

Reply:
Less concerned about readability, more thinking of actionability. Does a response to a password update have any chance of getting mistaken for a response to a name update? It may not...

Reply:
Below is where we are checking the status now. Below we are assuming either could have changed.  I can go either way if we need to create two statuses. 
```
    this.store.select(updateStatus).pipe(
      takeUntil(this.isDestroyed),
      filter(status => this.saveInProgress && !loading(status)))
      .subscribe((status) => {
        this.saveInProgress = false;
        // same status is used for updating password or display name, so we just reset both
        this.saveSuccessful = (status === EntityStatus.loadingSuccess);
        if (this.saveSuccessful) {
          this.resetForms();
        }
      });
```

Reply:
Looks fine as is; thanks for checking.